src/main/build/debug.c

```diff
@@ -50,6 +50,7 @@ const char * const debugModeNames[DEBUG_COUNT] = {
     "FFT_TIME",
     "FFT_FREQ",
     "RX_FRSKY_SPI",
+    "RX_SIGNAL_LOSS",    
```

Comment:
Adding the debug mode in the middle will break all blackbox viewers not updated to support this and using one of the others debug modes.